### PR TITLE
Document exportTo restrictions

### PR DESCRIPTION
**Pull request by frankbu**

### CODE REVIEW

File: networking/v1alpha3/destination_rule.proto [hunk 1]
```diff
@@ -118,9 +118,23 @@ message DestinationRule {
   // service. Traffic policies can be overridden at subset level.
   repeated Subset subsets = 3;
 
-  // The resolution of a DestinationRule to apply to a service occurs in the
-  // context of a hierarchy of namespaces. This rule controls whether those
-  // namespaces are allowed to select this rule.
+  // A list of namespaces to which this destination rule is exported.
+  // The resolution of a destination rule to apply to a service occurs in the
+  // context of a hierarchy of namespaces. Exporting a desintation rule allows
```
Comment (andraxylia) [inline networking/v1alpha3/destination_rule.proto]: ` // context of a hierarchy of namespaces. Exporting a destination rule allows`

File: networking/v1alpha3/destination_rule.proto [hunk 2]
```diff
@@ -118,9 +118,23 @@ message DestinationRule {
...
+  // across namespace boundaries.
+  //
+  // If no namespaces are specified then the destination rule is exported to all
+  // namespaces by default.
+  //
+  // The value "." is reserved and defines an export to the same namespace that
+  // the destination rule is declared in. Similarly, the value "*" is reserved and
+  // defines an export to all namespaces.
+  //
+  // NOTE: in the current release, the `exportTo` value is restricted to
```
Comment (andraxylia) [inline networking/v1alpha3/destination_rule.proto]:
```suggestion
  // NOTE: in the current release, the `export_to` value is restricted to
```

File: networking/v1alpha3/service_entry.proto [hunk 3]
```diff
@@ -480,12 +480,15 @@ message ServiceEntry {
...
-  // the service is declared in, similarly the value "*" is reserved and
+  // the service is declared in. Similarly, the value "*" is reserved and
   // defines an export to all namespaces.
   //
-  // For a Kubernetes Service the equivalent effect can be achieved by setting
+  // For a Kubernetes Service, the equivalent effect can be achieved by setting
   // the annotation "networking.istio.io/exportTo" to a comma-separated list
   // of namespace names.
+  //
+  // NOTE: in the current release, the `exportTo` value is restricted to
```
Comment (andraxylia) [inline networking/v1alpha3/service_entry.proto]:
```suggestion
  // NOTE: in the current release, the `export_to` value is restricted to
```

File: networking/v1alpha3/virtual_service.proto [hunk 4]
```diff
@@ -186,8 +186,11 @@ message VirtualService {
   // namespaces by default.
   //
   // The value "." is reserved and defines an export to the same namespace that
-  // the virtual service is declared in, similarly the value "*" is reserved and
+  // the virtual service is declared in. Similarly, the value "*" is reserved and
   // defines an export to all namespaces.
+  //
+  // NOTE: in the current release, the `exportTo` value is restricted to
```
Comment (andraxylia) [inline networking/v1alpha3/virtual_service.proto]:
```suggestion
  // NOTE: in the current release, the `export_to` value is restricted to
```

File: networking/v1alpha3/destination_rule.proto (hunk 2 above)
Comment (frankbu) [reply networking/v1alpha3/destination_rule.proto]: We need to use the camelCase in comments so they look right in the generated doc.

**Comment by frankbu** [comment]
Merged the wrong branch. Closing this one and opening new PR.

File: networking/v1alpha3/destination_rule.proto (hunk 2 above)
Comment (andraxylia) [reply networking/v1alpha3/destination_rule.proto]: ack

**Pull request closed without merging**